spec/rubocop/cli/autocorrect_spec.rb

```diff
@@ -2763,4 +2763,60 @@ module Foo#{trailing_whitespace}
       end
     RUBY
   end
+
+  it 'indents hashes correctly when adding a line break' do
+    source_file = Pathname('example.rb')
+    create_file(source_file, <<~RUBY)
+      def example
+        { key1:
+          { key2:
+            { key3:
+              { key4:
+                { key5:
+                  :value
+                }
+              }
+            }
+          }
+        }
+      end
+    RUBY
+
+    create_file('.rubocop.yml', <<~YAML)
+      AllCops:
+        DisabledByDefault: true
+      Style/FrozenStringLiteralComment:
+        Enabled: false
+
+      Layout/FirstHashElementIndentation:
+        Enabled: true
+      Layout/FirstHashElementLineBreak:
+        Enabled: true
+      Layout/TrailingWhitespace:
+        Enabled: true
+    YAML
+
+    status = cli.run(%w[--autocorrect-all])
```

Comment:
You should probably just specify (with `--only`, see line 2758 for an example) which cops you want to run for this test, and then you wouldn't need to explicitly enable/disable cops in the yaml file.